samples/dlp_inspect_string.js

```
@@ -0,0 +1,69 @@
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(
+  projectId = 'YOUR_PROJECT_ID',
+  stringToInspect = 'My name is Gary and my email is gary@example.com'
+) {
+  // [START dlp_inspect_string]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'YOUR_PROJECT_ID';
+  // const stringToInspect = 'My name is Gary and my email is gary@example.com';
+
+  // Imports the Google Cloud Data Loss Prevention library
+  const DLP = require('@google-cloud/dlp');
+
+  // Instantiates a client
+  const dlp = new DLP.DlpServiceClient();
+
+  async function inspectString() {
+    // Construct request
+    const request = {
+      parent: dlp.projectPath(projectId),
+      item: {
+        value: stringToInspect,
+      },
+      inspectConfig: {
+        // The infoTypes of information to match
+        infoTypes: [
+          {name: 'PHONE_NUMBER'},
+          {name: 'EMAIL_ADDRESS'},
+          {name: 'CREDIT_CARD_NUMBER'},
+        ],
+        // Whether to include the matching string
+        includeQuote: true,
+      },
+    };
+
+    // Run request
+    const [inspectResponse] = await dlp.inspectContent(request);
+    const findings = inspectResponse.result.findings;
+    console.log(`Findings: ${!findings.length ? 'None' : ''}`);
+    findings.forEach(finding => {
+      console.log(`\tQuote: ${finding.quote}`);
+      console.log(`\tInfo type: ${finding.infoType.name}`);
+      console.log(`\tLikelihood: ${finding.likelihood}\n`);
+    });
+  }
+
+  inspectString(projectId, stringToInspect).catch(console.error);
```

Why the catch?  Shouldn't we just let it throw?

---

Removed